yaml/pois.yaml

```diff
@@ -1,16 +1,424 @@
+      END
+  - &us_forest_service
+        - United States Forest Service
+        - US Forest Service
+        - U.S. Forest Service
+        - USDA Forest Service
+        - United States Department of Agriculture
+        - US National Forest Service
+        - United State Forest Service
+        - U.S. National Forest Service
+  - &us_parks_service
+        - United States National Park Service
+        - National Park Service
+        - US National Park Service
+        - U.S. National Park Service
+        - US National Park service
 filters:
   - filter: {disused: -no}
     min_zoom: null
     output:
       kind: {expr: null}
+
+  ############################################################
+  # NOT IN ANY TIER
+  #
+  # These depend on being run before the rule for tourism=attraction,
+  # otherwise that rule will trigger first. So these have been moved
+  # up here.
+  ############################################################
+  - filter: {man_made: windmill}
+    min_zoom: CASE WHEN tags ? 'tourism' AND tags->'tourism' = 'attraction' THEN 14
+      ELSE 15 END
+    output: {kind: windmill}
+  - filter: {man_made: lighthouse}
+    min_zoom: CASE WHEN tags ? 'tourism' AND tags->'tourism' = 'attraction' THEN 14
+      ELSE 15 END
+    output: {kind: lighthouse}
+  - filter: {man_made: observatory}
+    min_zoom: 15
+    output: {kind: observatory}
+
+  ############################################################
+  # TIER 1
+  ############################################################
+  - filter:
+      historic: battlefield
+      not: {operator: *us_forest_service}
+    min_zoom: { min: [ { max: [ 10, { lit: zoom + 4 }, { lit: *tier1_min_zoom } ] }, 17 ] }
+    output: {kind: battlefield, tier: 1}
+  - filter:
+      boundary: national_park
+      not: {operator: *us_forest_service}
+    min_zoom: { min: [ { max: [ { lit: zoom + 3.5 }, { lit: *tier1_min_zoom } ] }, 10 ] }
+    output:
+      kind: national_park
+      tier: 1
+
+  ############################################################
+  # TIER 2
+  ############################################################
+  - filter:
+      boundary: national_park
+      not: {operator: *us_parks_service}
+    min_zoom: { min: [ { max: [ { lit: zoom + 3.5 }, { lit: *tier2_min_zoom } ] }, 10 ] }
+    output:
+      kind: national_park
+      tier: 2
+  - filter:
+      any:
+        leisure: park
+        landuse: park
+      not: { operator: *us_parks_service }
+    min_zoom: { min: [ { max: [ 9, { lit: zoom + 2 }, { lit: *tier2_min_zoom } ] }, 14 ] }
+    output:
+      kind: park
+      tier: 2
+
+  # forests
+  - filter:
+      landuse: forest
+      protect_class: '6'
+    min_zoom: { min: [ { max: [ 7, { lit: zoom + 2 }, { lit: *tier2_min_zoom } ] }, 14 ] }
+    output: { kind: forest, tier: 2 }
+  - filter:
+      landuse: forest
+      operator: *us_forest_service
+    min_zoom: { min: [ { max: [ 9, { lit: zoom + 2 }, { lit: *tier2_min_zoom } ] }, 14 ] }
```

Comment:
If the operator is `*us_forest_service` then I think we'd want to see them at zoom 7 like the rule above?